pact/verify.py

```diff
@@ -46,6 +46,15 @@
 @click.option(
     'username', '--pact-broker-username',
     help='Username for Pact Broker basic authentication.')
+@click.option(
+    'broker_base_url', '--pact-broker-base-url',
```

Comment:
If I was writing the pact CLI again, I'd just call this "pact-broker-url". I don't think it needs the "base". How about we drop it in the python?